src/datasources/__tests__/grid.test.js

```diff
@@ -0,0 +1,284 @@
+import nock from 'nock'
+
+import gridDatasource from '../grid'
+
+let requestInfo
+
+describe('grid', () => {
+  const OLD_ENV = process.env
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+
+    jest.restoreAllMocks()
+
+    process.env = { ...OLD_ENV }
+
+    process.env.cmrRootUrl = 'http://example.com'
```

Comment:
if you happen to be in here for another change, make this example.gov because that is closer to reality.